projects/js-packages/licensing/components/activation-screen/index.jsx

```diff
@@ -65,6 +65,30 @@ const ActivationScreen = props => {
 	const [ isSaving, setIsSaving ] = useState( false );
 	const [ activatedProduct, setActivatedProduct ] = useState( null );
 
+	const [ fetchingAvailableLicenses, setFetchingAvailableLicenses ] = useState( false );
+	const [ availableLicenses, setAvailableLicenses ] = useState( [] );
+
+	useEffect( () => {
+		const { apiRoot, apiNonce } = window?.myJetpackRest || {};
+		restApi.setApiRoot( apiRoot );
+		restApi.setApiNonce( apiNonce );
+
+		setFetchingAvailableLicenses( true );
+		return restApi
+			.getUserLicenses()
+			.then( ( { items } ) => {
+				const detachedLicenses = items
+					? items.filter( ( { attached_at } ) => attached_at === null )
+					: [];
```

Comment:
I see this error here because the REST API seems to return an object with numeric indices instead of an array. Little big of debugging tells me that it's something that comes from WPCOM API inside `projects/packages/licensing/src/class-endpoints.php` in `Endpoints::get_user_licenses()`

<img width="1138" alt="Screenshot 2022-11-28 at 11 57 49 AM" src="https://user-images.githubusercontent.com/18226415/204208694-1a470816-d0b7-47c4-9fe3-7ae5302020b9.png">

Reply:
Created a patch to fix the bug on the backend D93569-code

Reply:
thanks for the quick fix @manzoorwanijk 👍